scripts/zones/Arrapago_Reef/TextIDs.lua

```diff
@@ -1,23 +1,29 @@
 -- Variable TextID   Description text
 
 -- General Texts
-ITEM_CANNOT_BE_OBTAINED = 6381; -- You cannot obtain the item <item> come back again after sorting your inventory
-          ITEM_OBTAINED = 6387; -- Obtained: <item>
-           GIL_OBTAINED = 6388; -- Obtained <number> gil
-       KEYITEM_OBTAINED = 6390; -- Obtained key item: <keyitem>
- FISHING_MESSAGE_OFFSET = 7048; -- You can't fish here
+ITEM_CANNOT_BE_OBTAINED =  6381; -- You cannot obtain the item <item> come back again after sorting your inventory
+          ITEM_OBTAINED =  6387; -- Obtained: <item>
+           GIL_OBTAINED =  6388; -- Obtained <number> gil
+       KEYITEM_OBTAINED =  6390; -- Obtained key item: <keyitem>
+ FISHING_MESSAGE_OFFSET =  7048; -- You can't fish here
 
 -- Assault
-           CANNOT_ENTER = 8444; -- You cannot enter at this time.  Please wait a while before trying again.
-              AREA_FULL = 8445; -- This area is fully occupied. You were unable to enter.
-         MEMBER_NO_REQS = 8449; -- Not all of your party members meet the requirements for this objective.  Unable to enter area.
-         MEMBER_TOO_FAR = 8453; -- One or more party members are too far away from the entrance.  Unable to enter area.
+           CANNOT_ENTER =  8444; -- You cannot enter at this time.  Please wait a while before trying again.
+              AREA_FULL =  8445; -- This area is fully occupied. You were unable to enter.
+         MEMBER_NO_REQS =  8449; -- Not all of your party members meet the requirements for this objective.  Unable to enter area.
+         MEMBER_TOO_FAR =  8453; -- One or more party members are too far away from the entrance.  Unable to enter area.
 
 -- Other Texts
-            YOU_NO_REQS = 7894; -- You do not meet the requirements to enter the battlefield
-        NOTHING_HAPPENS =  119; -- Nothing happens...
-               RESPONSE = 7328; -- There is no response...
+            YOU_NO_REQS =  7894; -- You do not meet the requirements to enter the battlefield
+        NOTHING_HAPPENS =   119; -- Nothing happens...
+               RESPONSE =  7328; -- There is no response...
+         DOOR_IS_LOCKED = 40225; -- The door is locked. You might be able to open it with a <item>.
```

Comment:
I see this as 7457 in the POLUtils extract

Reply:
I didn't use POLUtils, I got numbers from retail capture

Reply:
Hm, interesting.  Looks like the 7457 value is needed for player:messageSpecial(), whereas the 40225 value is needed for player:showText().  I wonder if these showText values will bonk the TextIDs batch for version update.  Will ask in Discord.

Reply:
Showtext is just a wrapper for message special that includes lookAt() as well. It simulates a startEvent() for a one liner

Reply:
I actually had to use messageSpecial *AND* showText in the script with different messages to get it to display properly without the player's name tag indicating who is speaking. ( "Damisha: " )
I don't know if those functions are set up that way on purpose but otherwise the text would show up as if the player was speaking it.

Reply:
There are some quirks I'll do my best to explain here:

if the calling entity is not a player, the calling entity will be the "speaker" for show text and it'll be an "anybody in range" message
```cpp
((CCharEntity*)m_PBaseEntity)->pushPacket(new CMessageSpecialPacket(PBaseEntity, messageID, param0, param1, param2, param3));
```
but if the calling entity is a player, then it is 
```cpp
PBaseEntity->loc.p.rotation = getangle(PBaseEntity->loc.p, m_PBaseEntity->loc.p);
```
(rotation and angle)
followed by literally just message special but with teh player (base entity) as the speaking entity.
```cpp
m_PBaseEntity->loc.zone->PushPacket(m_PBaseEntity, CHAR_INRANGE, new CMessageSpecialPacket(PBaseEntity, messageID, param0, param1, param3));
```
So the players name was there because the packet is making the player "speak" and that is probably not the correct way to get the message out (those things are usually an event or a parameter passed into a message or event) but sometimes we don't get a choice.

Reply:
In any case which copy of a duplicate message you use won't matter in darkstar.